repose-aggregator/services/service-client/api/src/main/java/org/openrepose/core/services/serviceclient/akka/AkkaServiceClient.java

```diff
@@ -29,6 +29,8 @@
  */
 public interface AkkaServiceClient {
 
+    void setConnectionPoolId(String connectionPoolId);
```

Comment:
It would break MaaS' custom filter.  How would we communicate that adequately?

https://github.com/rackerlabs/cloud-monitoring-filter-bundle/blob/master/extract-device-id/src/main/java/org/openrepose/filters/custom/extractdeviceid/ExtractDeviceIdFilter.java#L88